    public final int \u02cf()
    {
        java.lang.Throwable $r2;
        java.lang.NullPointerException r3;
        java.util.Vector $r1;
        util.h.xy.cy.rc r0;
        int $i0, $i1, $i2;
        boolean $z0;

        r0 := @this: util.h.xy.cy.rc;

        $i0 = <util.h.xy.cy.rc: int \u02ce>;

        $i1 = $i0 ^ 37;

        $i0 = $i0 & 37;

        $i0 = $i0 << 1;

        $i1 = $i1 + $i0;

        $i1 = $i1 % 128;

        <util.h.xy.cy.rc: int \u02ca> = $i1;

        $r1 = r0.<util.h.xy.cy.rc: java.util.Vector \u02cb>;

        $i1 = virtualinvoke $r1.<java.util.AbstractCollection: int size()>();

        $i0 = <util.h.xy.cy.rc: int \u02ca>;

        $i2 = $i0 + 89;

        $i0 = $i2 % 128;

        <util.h.xy.cy.rc: int \u02ce> = $i0;

        $i0 = $i2 % 2;

        if $i0 != 0 goto label1;

        $z0 = 1;

        goto label2;

     label1:
        $z0 = 0;

     label2:
        if $z0 == 1 goto label3;

        return $i1;

     label3:
        r3 = new java.lang.NullPointerException;

        specialinvoke r3.<java.lang.NullPointerException: void <init>(java.lang.String)>("This statement would have triggered an Exception: specialinvoke $u1#13.<java.lang.Object: int hashCode()>()");

        throw r3;

     label4:
        $r2 := @caughtexception;

        throw $r2;

        catch java.lang.Throwable from label3 to label4 with label4;
    }
